app/services/analytics_events.rb

```diff
@@ -611,8 +611,9 @@ def fraud_review_rejected(
   # @param [String] name
   # @param [String] message
   # @param [String] stack
-  def frontend_error(name:, message:, stack: nil, **_extra)
-    track_event('Frontend Error', name:, message:, stack:)
+  # @param [String] filename
+  def frontend_error(name:, message:, stack: nil, filename: nil, **_extra)
+    track_event('Frontend Error', name:, message:, stack:, filename:)
```

Comment:
I just noticed this isn't referencing `extra`--do we need to include it here? Actually... I think it is probably fine not to include it, and anyway it's not a change introduced in this PR

Reply:
Actually, now that you mention it, I don't think this method is actually used, since the [mapping](https://github.com/18F/identity-idp/blob/22aa3b1a13357be1153d7fa6c50f5b07043b8b3e/app/controllers/frontend_log_controller.rb#L14) in the frontend log controller calls `FrontendErrorLogger` directly. I also confirmed there are none of these events in CloudWatch for the past 2 weeks.

I'll go ahead and remove it.

Reply:
Removed in 8dcc1af2f.